installer/install-deno.cmd

```diff
@@ -0,0 +1,6 @@
+@echo off
+
+setlocal
+curl -L -o deno-x86_64-pc-windows-msvc.zip "https://github.com/hashicorp/terraform-ls/releases/latest/download/deno-x86_64-pc-windows-msvc.zip"
```

Comment:
always download `latest` version.

Reply:
`hashicorp/terraform-ls`? You may need to make changes to `denoland/deno`.

Reply:
oops 😨 
Thanks, fixed 🙇